components/hab/src/command/origin/secret/upload.rs

```diff
@@ -48,7 +48,7 @@ pub fn start(
         Err(_) => {
             return Err(Error::ArgumentError(
                 "Failed to convert encrypted bytes to string",
-            ))
+            ));
```

Comment:
As long as we're changing code here, we should make this
```rust
    let encrypted_secret_string =
        String::from_utf8(encrypted_secret_bytes).expect("base64-encoded ciphertext");
```
All the error handling is unnecessary and frankly misleading. See https://github.com/habitat-sh/core/issues/15 for background. This motivates me to actually go fix it. I'll do that soon as I can.